template/web/app/account-abstraction/index.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+
+import Banner from '@/components/banner/banner';
+import Footer from '@/components/footer/Footer';
+import Header from '@/components/header/Header';
```

Comment:
rebase with main these have been moved
```suggestion
import Banner from '@/components/layout/banner/banner';
import Footer from '@/components/layout/footer/Footer';
import Header from '@/components/layout/header/Header';
```

Reply:
Done